psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -1200,6 +1269,7 @@ SharedMatrix MintsHelper::mo_eri_helper(SharedMatrix Iso, SharedMatrix Co, Share
     auto Imo = std::make_shared<Matrix>("MO ERI Tensor", nocc * nvir, nocc * nvir);
     double **Imop = Imo->pointer();
 
+#pragma omp parallel for collapse(4) num_threads(nthread_)
```

Comment:
This loop is just doing assignment. I doubt it is CPU bound. Parallellization might actually hurt performance.